lib/mumukit/core/class.rb

```diff
@@ -0,0 +1,20 @@
+class Class
+  class ContractBreachError < Exception
+  end
+
+  class AbstractClassInstantiationError < Exception
+  end
+
+  unless %w(RACK_ENV RAILS_ENV).any? { |it| ENV[it] == 'production' }
```

Comment:
I think we need something more general like a `Mumukit::Core.test_mode?` and a setter `Mumukit::Core.test_mode!`. 

That property should be set by default with `%w(RACK_ENV RAILS_ENV).any? { |it| ENV[it] == 'production' }`.

That way, this code will property work even on non-rack environments, and we will be able to force this mode in other environments if needed.

Reply:
I was just thinking... we can't have `test_mode!` simply set a flag to true, since this condition is evaluated as soon as this file is loaded, so something like:
```
require 'mumukit/core'
Mumukit::Core.test_mode!
```

would _not_ apply this patch.

What I guess we _could_ do was to make this patch part of the `test_mode!` method logic